package NKikimrCountersInfoProto;

message TEvCountersInfoRequest {
}

message TEvCountersInfoResponse {
    optional string Response = 1;
}
